tests/integration/tools/example/pom.xml

```diff
@@ -38,7 +38,7 @@
     <name>Helidon Integration Tests Tools: Test Example</name>
 
     <properties>
-        <version.lib.jandex-maven-plugin>1.0.6</version.lib.jandex-maven-plugin>
+        <version.lib.jandex-maven-plugin>3.1.1</version.lib.jandex-maven-plugin>
```

Comment:
Ding dong version upgrade calling

Reply:
Fixed. I've searched for all occurrences of "3.1.1" and none still refer to Jandex.